lib/screens/forgot_password/forgot_password.dart

```diff
@@ -36,41 +40,49 @@ class _ForgotPasswordState extends State<ForgotPassword> {
 
   @override
   Widget build(BuildContext context) {
-    final text = AppLocalizations.of(context);
-
+    final text = S.of(context);
+    final size = MediaQuery.of(context).size;
     return Scaffold(
-      appBar: AppBar(),
-      backgroundColor: CustomColors().background,
-      body: SingleChildScrollView(
-        child: SafeArea(
-          minimum: EdgeInsets.symmetric(horizontal: 20),
-          child: Container(
-            padding: EdgeInsets.only(top: 140),
-            child: Column(
-              children: [
-                Text(
-                  text!.recover_password,
-                  style: TextStyle(
-                    fontSize: 37,
-                    fontWeight: FontWeight.bold,
+      body: Container(
+        width: double.infinity,
```

Comment:
What if you wrap the widget inside an Expanded or Flexible widget, I think that it could avoid using the double.infinity